grammars/prql-lezer/src/prql.grammar

```diff
@@ -34,10 +34,10 @@ tupleItem { AssignTupleItem | expression | CallExpression | CaseBranch }
 // being parsed as a CallExpression, e.g. `s"{a"` -> `s` & `"{a"'. But we
 // can't seem to force a space because it's in our skip, and I can't see
 // a way of changing the skip expression to only specialize on a single item
-CallExpression { Identifier ArgList { (NamedArg | Assign | test)+ } }
+CallExpression { Identifier ArgList { (NamedArg | Parameter | test)+ } }
 
-NamedArg { identPart ":" expression }
-Assign { identPart "=" expression }
+NamedArg { ArgumentName { identPart } ":" expression }
+Parameter { ParameterName { identPart } "=" expression }
 AssignTupleItem { PropertyName { identPart } "=" expression }
```

Comment:
TBC, these two are very very similar and should be formatted exactly the same, so very likely `ParameterName` and `PropertyName` should be the same...

```
Parameter { ParameterName { identPart } "=" expression }
AssignTupleItem { PropertyName { identPart } "=" expression }
```

Reply:
And `Parameter` & `AssignTupleItem` should be very similar — I think both named `Assign` in some way

Reply:
You earlier said we don't have properties, so do you want both both `Parameter` and `AssignTupleItem` to have a child token named `ParameterName`. I am not sure `ParameterName` is a good name for the tuple item since parameters are for f unctions, not for tuples.

Also, I don't like the name "assign" because it is a verb.

So maybe `AssignTupleItem` should just be called `TupleItem`?

Reply:
But it's not any `TupleItem` — it only applies to assignments.

---

I don't want to dissuade you from making small incremental changes — they often add up to great changes! And overall you've made lots of great contributions to this repo.

But I think this one may not be worthwhile — it must be taking up a decent amount of your time, and it has been the most commented issue from me in the past week. Unless we have a way that is obviously better, I might suggest we leave this for a while, see how other changes go, and we could come back to it later. WDYT?

Reply:
Then maybe it is a `TupleItemDeclaration`?

Well, this one enables highlighting of that token. Maybe you could just say some names, and we'll use that, we can always change it later.

Reply:
> Then maybe it is a `TupleItemDeclaration`?

Yes this works! So we have something like

```
- Assign { identPart "=" expression }
+ Declaration { DeclarationItem { identPart } "=" expression }`
- AssignCall { identPart "=" expression }
+ DeclarationTuple { DeclarationItem { identPart } "=" expression }
```

`DeclarationItem` could also be `Variable` or `Alias`. I think it should be the same in both so they're formatted the same

WDYT?